ankimani/StackOverflow-lite
# #162560558 Added Post Qustions UI

#### What does this PR do?
It will Add UI for users to post Questions
#### Description of Task to be completed?
The Following Tasks Should be completed:
1.Create UI that allows users to  Post Answers to Questions
2. Add styling to the Post Questions UI
#### How should this be manually tested?
This can be tested as follows:
1. 0n git via https://github.com/ankimani/StackOverflow-lit Clone the Repository to your desired folder in your local repository 
2.Then Open the file askquestion.html using your desired browser and check whether the contents of the UI displays Properly in the browser. Also test for Responsivess by minimising the browser to test display on different devices screen
#### What are the relevant pivotal tracker stories?
#162560558
